Mathlib/Algebra/Group/Units/Opposite.lean

```diff
@@ -0,0 +1,57 @@
+/-
+Copyright (c) 2018 Kenny Lau. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kenny Lau
+-/
+import Mathlib.Algebra.Group.Opposite
+import Mathlib.Algebra.Group.Units.Defs
```

Comment:
```suggestion
/-
Copyright (c) 2021 Eric Wieser. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Eric Wieser
-/
import Mathlib.Algebra.Group.Opposite
import Mathlib.Algebra.Group.Units.Defs
```
for https://github.com/leanprover-community/mathlib3/commit/a85fbdaed80c3d086acf3a0c6276e12829d977d2 and https://github.com/leanprover-community/mathlib3/commit/0372d31fb681ef40a687506bc5870fd55ebc8bb9